packages/fast-components-styles-msft/src/checkbox/index.ts

```diff
@@ -56,90 +72,69 @@ const styles: ComponentStyles<CheckboxClassNameContract, DesignSystem> = (
-                border: `${toPx(
-                    designSystem.outlinePatternOutlineWidth
-                )} solid ${neutralFocus(designSystem)}`,
+                borderColor: neutralFocus(designSystem),
             }),
         },
         checkbox_stateIndicator: {
             position: "relative",
-            borderRadius: "2px",
+            ...applyCornerRadius(designSystem),
             display: "inline-block",
-            width: "20px",
-            height: "20px",
+            width: toPx(size),
+            height: toPx(size),
             flexShrink: "0",
-            "&::before, &::after": {
+            "&::before": {
                 content: "''",
                 pointerEvents: "none",
-                width: "2px",
                 position: "absolute",
                 zIndex: "1",
-            },
-            "&::before": {
-                top: "4px",
-                left: "11px",
-                height: "12px",
-                transform: "rotate(40deg)",
-            },
-            "&::after": {
-                top: "9px",
-                left: "6px",
-                height: "6px",
-                transform: "rotate(-45deg)",
+                top: "0",
+                left: "0",
+                width: toPx(size),
+                height: toPx(size),
             },
         },
         checkbox_label: {
             color: neutralForegroundRest,
-            ...applyTypeRampConfig("t7"),
+            ...applyFontSize(designSystem),
         },
         checkbox__checked: {
             "& $checkbox_stateIndicator": {
-                "&::after, &::before": {
-                    position: "absolute",
-                    zIndex: "1",
-                    content: '""',
-                    borderRadius: toPx(designSystem.cornerRadius),
-                    background: neutralForegroundRest,
+                "&::before": {
+                    background: `url('data:image/svg+xml;utf8,${indicatorSvg}')`,
                     "@media (-ms-high-contrast:active)": {
-                        backgroundColor: "ButtonHighlight",
+                        background: `url('data:image/svg+xml;utf8,${indicatorSvgHc}')`,
                     },
                 },
             },
         },
         checkbox__indeterminate: {
             "& $checkbox_stateIndicator": {
                 "&::before": {
-                    borderRadius: toPx(designSystem.cornerRadius),
+                    ...applyCornerRadius(designSystem),
                     transform: "none",
-                    [applyLocalizedProperty("left", "right", direction)]: "5px",
-                    top: "5px",
-                    height: "10px",
-                    width: "10px",
+                    [applyLocalizedProperty("left", "right", direction)]: indicatorMargin,
+                    top: indicatorMargin,
+                    bottom: indicatorMargin,
+                    right: indicatorMargin,
```

Comment:
should you be setting this here? seems like the applyLocalizedProperty above should set left/right